tests/unit/bentoml_io/test_decorators.py

```diff
@@ -4,8 +4,8 @@
 import numpy.typing as npt
 import pandas as pd
 import pytest
-from typing_extensions import Annotated
 from starlette.testclient import TestClient
+from typing_extensions import Annotated
```

Comment:
doesn't seem like this change is related?

Reply:
No, but it's just an import reorder.